src/core/packages/plugins/server-internal/src/discovery/plugins_discovery.ts

```diff
@@ -71,6 +71,14 @@ export function discover({
         parentDirs: config.pluginSearchPaths,
       })
     ),
+    filter(
+      // only admit those packages that ...
+      (pkg) =>
+        // ... are NOT under an excluded group
+        !config.excludedPluginGroups.includes(pkg.getGroup()) ||
+        // ... or that are under an explicitly included group
+        config.includedPluginGroups.includes(pkg.getGroup())
+    ),
```

Comment:
🤔 at the moment this `||` logic seems a bit too permissive... I understand it works right now bc we have an exhaustive list of groups to exclude by default, but it seems that whenever `excludedPluginGroups = []` we will allow everything (regardless of what includes contains).